man/en/sos-collect.1

```diff
@@ -350,6 +351,20 @@ Note that sosreports are collected in parallel, so you can approximate the total
 runtime of sos collect via timeout*(number of nodes/jobs).
 
 Default is 180 seconds.
+.TP
+\fB\-\-transport\fR TRANSPORT
+Specify the type of remote transport to use to manage connections to remote nodes.
+
+\fBsos collect\fR uses locally installed binaries to connect to and interact with remote
+nodes, instead of directly establishing those connections. By default, OpenSSH's ControlPersist
+feature is preferred, however certain cluster types may have preferences of their own for how
+remote sessions should be established.
+
+The types of transports supported are currently as follows:
+
+    \fBauto\fR                  Allow the cluster type to determine the transport used
+    \fBcontrol_persist\fR       Use OpenSSH's ControlPersist feature. This is the default behavior
```

Comment:
Subjective opinion: From the type name `control_persist`, I would not deduce we use `ssh` for the transport. Could "ssh" be part of the name? (But `ssh_control_persist` is too long, `ssh` might lack the ControlPersist feature info (is the info in type name required?)]

Reply:
There's the possibility that in the future we could add a transport that uses, for example, the libssh python bindings. In which case we need an obvious way to differentiate between the two, but I couldn't think of anything besides this or as you mentioned `ssh_control_persist` which is a bit heavy.

Reply:
I understand. `ssh` is too generic, `ssh_control_persist` too long. So let use the currently proposed term.